regression/cbmc/sync_X_and_fetch-1/main.c

```diff
@@ -0,0 +1,16 @@
+#include <assert.h>
+
+int main()
+{
+#ifndef _WIN32
```

Comment:
Consider use of a tag in the .desc file rather than an ifndef (it's clearer if on Win32 the test reads SKIPPED)

Reply:
Done.